hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/impl/ITestPartialRenamesDeletes.java

```diff
@@ -54,7 +54,12 @@
 import static org.apache.hadoop.fs.contract.ContractTestUtils.*;
 import static org.apache.hadoop.fs.s3a.Constants.*;
 import static org.apache.hadoop.fs.s3a.S3ATestUtils.MetricDiff;
-import static org.apache.hadoop.fs.s3a.S3ATestUtils.*;
+import static org.apache.hadoop.fs.s3a.S3ATestUtils.assume;
```

Comment:
not sure the expansion here is needed..,you know how imports are such a backporting troublespot.

in fact, given there's no other diff to this class, I'm not sure this file needs changing at all

Reply:
I got issue on getting test bucket name even if `test.fs.s3a.name` was defined in the test conf and othter itests worked.
```
java.lang.NullPointerException: No test bucket
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at org.apache.hadoop.fs.s3a.S3ATestUtils.getTestBucketName(S3ATestUtils.java:739)
	at org.apache.hadoop.fs.s3a.impl.ITestPartialRenamesDeletes.createConfiguration(ITestPartialRenamesDeletes.java:326)
```

I suspected the configuration reloading in S3ATestUtils as the cause. The change is leftover of trial-and-error. Since I can not reproduce the issue now, I will revert this part.